docker-hub/api/latest.yaml

```diff
@@ -100,6 +100,10 @@ tags:
     description: |
       The repository endpoints allow you to manage your repository's
       configuration like description.
+  - name: scim
+    x-displayName: SCIM
+    description: |
+      Endpoints for SCIM integrations.
```

Comment:
SCIM is a provisioning system that lets you manage users within your identity provider (IdP).  For more information, see [System for cross-domain identity management](https://docs.docker.com/docker-hub/scim/).